src/linear_algebra/sesquilinear_form.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Andreas Swerdlow
 -/
 import algebra.module.linear_map
+import linear_algebra.basis.bilinear
 import linear_algebra.bilinear_map
```

Comment:
```suggestion
```

Reply:
I don't think this buys anything, and it makes it easier to move the basis stuff elsewhere again later.